### ISSUE TITLE
Post- 3.2.0 update-- Getting an Error from PowerPaimon that results in it crashing and the game running in 60fps.

### ISSUE THREAD

**Issue by MylesWow** [rexent-gx/PowerPaimon#3]
Running the game on a pc with the latest .NET installed+updated. Updated Genshin to 4.8, updated PowerPaimon to 3.2.0. I deleted the existing .cfg file. Unfortunately, PowerPaimon crashes and fails with the new Genshin version regardless-!

![image](https://github.com/user-attachments/assets/9e5c97f6-6e47-4658-b325-8b071526bc68)

![image](https://github.com/user-attachments/assets/ac69cf2a-8adc-459d-95a2-f08e413b3636)

PowerPaimon gives the error message:
"Failed to get remote module base address."

**Comment by catteol** [comment]
I tested PowerPaimon v3.2.0 with same Genshin client version on Windows 10 and 11 now, but it works properly.
So please (re)install [.NET Desktop Runtime 8.x.x](https://dotnet.microsoft.com/en-us/download/dotnet/8.0), or add to whitelist in your antivirus if you're using and try again.

if all else fails, uninstall genshin and launcher completely and reinstall it. I'm on HoYoPlay with basic install dir (C:\Program Files\HoYoPlay\games\Genshin Impact game).

**Comment by MylesWow** [comment]
Update: I wasn't using HoyoPlay to launch Genshin Impact, so I deleted the game and reinstalled from scratch with HoyoPlay just in case. Then, I uninstalled .NET using its installer, then reinstalled. Whitelisted the PowerPaimon .exe in Windows Security. Downloaded PowerPaimon latest and set up. Restarted pc== still getting the same error message from PowerPaimon.

![image](https://github.com/user-attachments/assets/c6df01a7-2809-457c-b6cd-70d22556ffaf)

![image](https://github.com/user-attachments/assets/f143f008-454a-4bc5-8f5c-ba570fd595bf)

![image](https://github.com/user-attachments/assets/7f57bff6-d5b5-4c75-92e2-a88212a9acb5)

**Comment by MylesWow** [comment]
Using Windows 11. Checked Windows Security and it's not blocking PowerPaimon.

**Comment by MylesWow** [comment]
My PP dir: E:\e-games\HoyoPlay\HoYoPlay\[tools]\PowerPaimon.exe
My game dir: E:\e-games\HoyoPlay\HoYoPlay\games\Genshin Impact game\GenshinImpact.exe

Not launching off of C:, which may be an issue. However, I don't have the space on my C drive for large games to test.. But the tool has worked fine with my E: drive setup for all previous versions of PP.

**Comment by MylesWow** [comment]
Another important piece of info is that the game's .exe seems to be launching separately from PowerPaimon

![image](https://github.com/user-attachments/assets/3fa60a77-609e-4274-a280-4fd45e9ca390)

PowerPaimon's process will come up separately with the crash dialogue once the login screen comes up: 

![image](https://github.com/user-attachments/assets/12991097-d66e-4061-bdd8-3d457a327bbc)

**Comment by catteol** [comment]
Thank you for sharing info.
Now I found some information that might be a solution. This error caused by old "UnityPlayer.dll" and "UserAssembly.dll" still remain in game folder. So please check if these 2 files exist in your game folder and remove them if they exist.
If this does not solve the problem, follow these steps:

1. Open the Hoyoplay, and select repair in the game option;
1. Delete fps_config.json of the PP;
1. Restart the PP and select the path of game again.

**Comment by MylesWow** [comment]
I searched for those two files in my new game folder, they weren't there. Ran the Hoyoplay launcher repair anyways== still not there. 

![image](https://github.com/user-attachments/assets/6c47716b-2957-42f4-93f1-a94be3335021)


![image](https://github.com/user-attachments/assets/2263b2ab-ea17-47fd-b889-307648fe3119)


I had already deleted and reinstalled all my game files under Hoyoplay, sorry if that wasn't clear.

But then, when I replaced PP again-- it suddenly worked. The only variable that has changed since my last attempt was using the HoyoPlay launcher to repair game files for the first time since the new install of the game. 

Maybe there were some missing files from the new game update that were only available from the file repair option. Thank you for the quick replies.
